crypto/ec/curve448/eddsa.c

```diff
@@ -0,0 +1,359 @@
+/*
+ * Copyright 2017-2018 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2015-2016 Cryptography Research, Inc.
+ *
+ * Licensed under the OpenSSL license (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ *
+ * Originally written by Mike Hamburg
+ */
+#include <openssl/crypto.h>
+#include <openssl/evp.h>
+
+#include "curve448_lcl.h"
+#include "word.h"
+#include "ed448.h"
+#include <string.h>
```

Comment:
I'd like to move this system header file a few places up.

Reply:
Done. I moved it to the top so we have system headers first; public openssl headers next; private openssl headers last.